### PR TITLE
Cleanup legacy/unused parts of the codebase: Venue configs, static assets, SparkleSpaceMarketingPage

### PR DESCRIPTION
**Pull request by 0xdevalias**
This was originally started in https://github.com/sparkletown/sparkle/pull/833, but as that had diverged too far from our current codebase, and to de-risk removing so many things at once + simplify code review, I decided to split this task into smaller PRs.

This PR removes:

- Legacy venue config `.json` files
- `SparkleSpaceMarketingPage` component + related components/assets
- Seemingly unused/legacy images/similar assets

**Note:** It seems we are using a bunch of asset files in a 'magic string' sort of way, which means that build tooling/IDEs may not properly detect what is 'unused'. I've made a best effort to check that none of the assets i'm removing here are used anywhere, but it's probably worth double checking this before we merge, as this could potentially break random images/etc across the site if we missed something.

### CODE REVIEW

**Review by denisdimitrov — DISMISSED** [review]
LGTM!

I didn't have the time to click through the UI and see if anything is broken. If you don't have the time to do it. Just leave a note and I'll test it first thing in the morning.

**Comment by 0xdevalias** [comment]
> I didn't have the time to click through the UI and see if anything is broken. If you don't have the time to do it. Just leave a note and I'll test it first thing in the morning.

@denisdimitrov I haven't done this yet. Will add another comment if I get around to doing so before you do, but if not, assume that I haven't got to it.

**Comment by denisdimitrov** [comment]
@0xdevalias Looks good. Feel free to merge it.

File: src/settings.ts [hunk 1]
```diff
@@ -35,16 +35,13 @@ export const PRIVACY_POLICY = IS_BURN
...
   : SPARKLE_PRIVACY_POLICY;
 
+export const SPARKLE_ICON = "/sparkle-icon.png";
 export const DEFAULT_MAP_BACKGROUND = "/maps/Sparkle_Field_Background.jpg";
 export const DEFAULT_VENUE_BANNER = "/assets/Sparkle_Banner_Default.jpg";
 export const DEFAULT_VENUE_LOGO = "/assets/Sparkle_SquareLogo_Default.jpg";
 export const DEFAULT_PROFILE_IMAGE = "/anonymous-profile-icon.jpeg";
 export const DEFAULT_AVATAR_IMAGE = "/icons/sparkle-nav-logo.png";
 export const DEFAULT_PARTY_NAME = "Anon";
-export const SPARKLEVERSE_LOGO_URL = "/sparkleverse-logo.png";
```
Comment (0xdevalias) [inline src/settings.ts:44]: This still seems like it is relevant to `src/pages/Account/Step6b.tsx:28`

**Review by 0xdevalias — COMMENTED** [review]
Manually searching for each of the deleted files in the codebase to ensure they aren't referenced anywhere.

The following files were deleted, but are still referenced, so need to be restored:

- `public/chat-icon.png`
  - by `src/components/templates/PartyMap/components/RoomCard/RoomCard.scss:117`
- `public/empty-chair.png`
  - by `src/components/organisms/Room/Room.tsx:283`
- `public/icons/chat-delete-icon.png`
  - by `src/components/molecules/ChatList/ChatList.scss:108`
- `public/icons/chat-input-submit.png`
  - by `src/components/molecules/Chatbox/Chatbox.scss:43`
- `public/navbar-link-gift.png`
  - by `src/components/molecules/NavBar/playa.scss:123`
- `public/navbar-link-radio-off.png`
  - by `src/components/molecules/NavBar/playa.scss:107`
- `public/navbar-link-radio.png`
  - by `src/components/molecules/NavBar/playa.scss:102`
- `public/onboarding-page/move.jpg`
  - by `src/pages/Account/Step6b.tsx:63`
- `public/onboarding-page/search.jpg`
  - by `src/pages/Account/Step6b.tsx:86`
- `public/onboarding-page/view.jpg`
  - by `src/pages/Account/Step6b.tsx:40`
- `public/sparkleverse-logo.png`
  - by `src/pages/Account/Step6b.tsx:28`

File: src/settings.ts (hunk 1 above)
Comment (0xdevalias) [reply src/settings.ts:44]: fixed in https://github.com/sparkletown/sparkle/pull/1120/commits/b7fd02de0a031b10779e5e6c327d1caefe070aba

**Comment by 0xdevalias** [comment]
@denisdimitrov This should be ready to go again now. Probably won't need the clickthrough again, and only a couple of tiny code changes + restoring some files that are still referenced, as per https://github.com/sparkletown/sparkle/pull/1120#pullrequestreview-582068665

**Review by denisdimitrov — APPROVED** [review]
LGTM!

**Pull request merged**